Source/WebCore/html/HTMLSelectElement.cpp

```diff
@@ -66,8 +66,8 @@ using namespace WTF::Unicode;
 
 using namespace HTMLNames;
 
-// Upper limit agreed upon with representatives of Opera and Mozilla.
-static const unsigned maxSelectItems = 10000;
+// https://html.spec.whatwg.org/#dom-htmloptionscollection-length
+static const unsigned maxSelectItems = 100000;
```

Comment:
`constexpr` is preferred.

Reply:
Will update after EWS run to confirm all tests are now passing so I can do rebase and other bits in one go. Thanks for review!

Reply:
Done.